index.bs

```diff
@@ -1943,53 +1947,88 @@ Constructors</h4>
         Sending a <a>control message</a> to start processing means
         executing the following steps:
     
-        <div class="addition proposed" id="c2400-5">
-        <span class="marker">Proposed Addition</span><br>
-            <a href="https://github.com/WebAudio/web-audio-api/issues/2400">Issue
-                2400</a> Access to a different output device
-        <div class="amendment-buttons"></div>
-        <del cite=#2400>
-            1. Attempt to <a href="#acquiring">acquire system resources</a>.
-                In case of failure, abort the following steps.
-
-            3. Set the {{[[rendering thread state]]}} to <code>running</code> on the {{AudioContext}}.
+        <div class="addition proposed">
+            <div class="addition proposed" id="c2400-5" style="border:0; margin:0; padding:0">
```

Comment:
See
Seems odd to have nested divs here?  
```suggestion
        <div class="addition proposed">
            <div class="addition proposed" id="c2400-5" style="border:0; margin:0; padding:0">
```

Reply:
Certain sections needed multiple "addition proposed" entries. This is the way to make it happen. (This is a new style/method that can only be seen in Web Audio API spec)

cc @svgeesus

Reply:
Oh, ok! that's super interesting... cc'ing @fantasai, as we are looking at how people use all this.

Reply:
Feel free to reopen if this is an issue! I would love to provide more feedback on this "proposed addition" markup.